lib/bibliothecary/dependency.rb

```diff
@@ -72,6 +72,10 @@ def eql?(other)
     end
     alias :== :eql?
 
+    def [](key)
+      public_send(key)
+    end
+
```

Comment:
This keeps backwards compatibility with code that _reads_ these attributes like a hash. Seemed friendlier than forcing the new syntax (and skips breaking some code on Libraries.io)